### ISSUE TITLE
Unset the automatic conversion from " to «

### ISSUE THREAD

**Issue by tubia** [msprev/dot-panzer#4]
Hi,  
I'm trying to understand how to disable the conversion of the `"` in `«` or `»`.
input files is a `.md` file, output is `.pdf` file.
I used `--smart` as an option but still conversion remains.

I'm posting here because I'm using your dot files :)

**Comment by msprev** [comment]
Pandoc has a quirk: it turns on `--smart` by default for the latex writer. This converts all your quotes. It's a quirk because `--smart` is a reader option, not a writer option. You can see a discussion of this here: 

https://groups.google.com/d/msg/pandoc-discuss/ZiHEil3E-ls/6HkPKIegCQAJ

Originally, I did not reproduce this quick in panzer, but users found that confusing, so I changed panzer to behave exactly as pandoc. 

The solution for both pandoc and panzer is to use the command line option `--no-tex-ligatures` for the latex writer. You can do this in a style definition as follows:

```
ExampleStyle:
  latex:
    commandline:
        smart: false
        no-tex-ligatures: true
```

**Comment by msprev** [comment]
If your output format is not latex, then simply remove any instances of the below from your style definition:

```
commandline:
  smart: true
```

Removing these (or setting `smart: false`) will turn off conversion of quotes.

**Comment by tubia** [comment]
Hi,
with `smart: false` all the " disappear, while with `no-tex-ligatures: true` nothing changes. Both results in no ".
Hints?

**Comment by msprev** [comment]
Hmm, the way to debug this is to try running vanilla pandoc on your document and identify which command line switches produce the output that you are looking for. If you can tell me those switches, then I can tell you want to put into the panzer style file. It is also possible that the template that you are using (one of mine?) doesn't support non-smart quotes -- this is something I never tested. Start with something small and simple with vanilla pandoc, and then build up from there.

**Comment by tubia** [comment]
Hi @msprev with vanilla pandoc it works (I used uberwriter to build but it simple execute pandoc). Checking the template in `/usr/share/pandoc/data/templates` named `default.latex` seems to be quite different in the packages it uses compared to the one in your dot files (that I'm using with panzer).
Maybe is something related to that?
This is the [`default.latex`](https://gist.github.com/tubia/3b3bd6c5956a629f5a7f) pandoc file

**Comment by msprev** [comment]
It must be the template. My templates were hacked together for my personal use (as the warning on the front page goes) and were not designed with this particular case in mind. You can modify my templates, or stick in your own, or stick in ones from uberwriter. Just copy a template file into the `/template/` director of `.panzer` and change the `template:` field in the style definition to source it.

It's on my todo list to refactor my templates to make them more flexible and cover the same cases as pandoc's default templates. This will happen, but not right now. I'd suggest that you hack your own template in the meantime. You can see pandoc's default latex template by running `pandoc -D latex`.

Sorry not to be more help on this but reworking a template is a major undertaking that, while on my horizon, is not likely to happen right now!

**Comment by tubia** [comment]
Hi,
I'll check the differences between default pandoc template - that works for me - and yours to see if I can make it works. I think your template could be a good starting point to beginners with panzer. This is the reason why I made a PR some days ago :)

**Comment by msprev** [comment]
Thanks for your kind words! Let me know if it doesn't work on switching to a new template. I really do need to tidy up my templates sometime so that these problems don't arise.

**Comment by tubia** [comment]
No problem!
Well actually I still don't understand the main differences between the files: to me differences are mainly for the `xelatex` and `luatex` part of the pandoc template file, but besides that I don't see anything regarding quotes.
Very strange.

**Comment by msprev** [comment]
I think it's likely that something is happening on the `xelatex` part that is causing your quotes problem.

**Comment by tubia** [comment]
Fixed.
[This](https://github.com/msprev/dot-panzer/blob/master/template/article.latex#L38) was the line incriminated. It use the package [csquotes](https://www.ctan.org/pkg/csquotes) which I don't have installed and that usually converts, with `autostyle` set, basing on the language declared using `babel` (or `polyglossia`). Removing the line resolves the problem. I'm interested however in re-implementing it to see if I can make it works basing on the language (but before that I need to sent a paper so I simply removed the line).
